client/src/components/SignUpForm.jsx

```diff
@@ -44,8 +47,10 @@ const SignUpForm = () => {
           });
           
             if (response.data.message === "User registered successfully") {
```

Comment:
"User registered successfully" try to move this magic string to a constant or a config